Source/doom.cpp

```diff
@@ -3,11 +3,7 @@
 int doom_quest_time;
 int doom_stars_drawn;
 BYTE *pDoomCel;
-#ifdef HELLFIRE
-BOOLEAN doomflag;
-#else
-BOOL doomflag;
-#endif
+HF_BOOL doomflag;

```

Comment:
This might actually be a good ida, not sure how the code purists will feel since it will mean the code will deviate a bit from the original

Reply:
I got a lot of cleanups coming up that use this, it's annoying to write 5 lines just to change BOOL into BOOLEAN, with this it's a matter of adding HF_ :)

Reply:
I say go for it :+1: